server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationRelocationIT.java

```diff
@@ -58,15 +60,20 @@ private void createIndex() {
     @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/issues/5669")
```

Comment:
does this still need to be muted?

Reply:
Yes @mch2 as the test occasionally fails due to doc count mismatch towards the end.

Reply:
https://github.com/opensearch-project/OpenSearch/pull/6015 should help.

Reply:
Thanks @mch2 for fixing flaky tests. The test is not flaky anymore. Ran 100 times without failure.